iopipe/iopipe.py

```diff
@@ -61,7 +61,7 @@ def error(self, error):
     def __call__(self, func):
         @functools.wraps(func)
         def wrapped(event, context):
-            logger.debug('%s.%s wrapped with IOpipe decorator' % (func.__module__, func.__name__))
+            logger.debug('%s wrapped with IOpipe decorator' % repr(func))
```

Comment:
The output of this now looks like:
`DEBUG:iopipe:<function hello at 0x7fe76063f950> wrapped with IOpipe decorator`
I'm not a big fan of this change for this DX modification. What's the representation when using Chalice vs a function (my example is with serverless framework)?

Reply:
It looks like this:

    DEBUG:iopipe:<chalice.app.Chalice object at 0x110235be0> wrapped with IOpipe decorator

The `func.__class__.__name__` type code I had before seemed uglier and more error prone than just getting the func's Python representation.

Reply:
Previous was definitely more error prone ... I liked the previous user-facing message better, but since this is logging output, it's not even necessarily seen as much, and better for it to be 100% reliable.